syncd/scripts/syncd_init_common.sh

```diff
@@ -367,7 +367,10 @@ config_syncd_xsight()
 
     CMD_ARGS+=" -p $HWSKU_DIR/sai.profile"
 }
-
```

Comment:
please keep empty lines before and after functions

Reply:
Hi kcudnik：
      The code has been modified to keep empty lines before and after functions,please help to review again,thank you !